.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Publish to Registry
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: banesullivan/pyvista
```

Comment:
I made CI to publish docker image. I found @banesullivan 's dockerhub page. It maybe prefer to make pyvista organization in Dockerhub.

Reply:
That docker image isn't great (I have a better one locally somewhere)... we should upload a better one under the PyVista org: https://hub.docker.com/u/pyvista

Reply:
Thank @banesullivan for comment. How about revert this CI and Dockerfile and split issue and pull request?